contracts-local/hardhat.config.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @type import('hardhat/config').HardhatUserConfig
+ */
```

Comment:
as discussed elsewhere, I don't think this is necessary but we can delete it in #3311 that's fine with me